packages/editor/src/components/block-mover/index.js

```diff
@@ -106,15 +117,17 @@ export class BlockMover extends Component {
 }
 
 export default compose(
-	withSelect( ( select, { clientIds, rootClientId } ) => {
-		const { getBlock, getBlockIndex, getTemplateLock } = select( 'core/editor' );
+	withSelect( ( select, { clientIds } ) => {
+		const { getBlock, getBlockIndex, getTemplateLock, getBlockRootClientId } = select( 'core/editor' );
 		const firstClientId = first( castArray( clientIds ) );
 		const block = getBlock( firstClientId );
+		const rootClientId = getBlockRootClientId( first( castArray( clientIds ) ) );
 
 		return {
 			firstIndex: getBlockIndex( firstClientId, rootClientId ),
 			blockType: block ? getBlockType( block.name ) : null,
 			isLocked: getTemplateLock( rootClientId ) === 'all',
+			rootClientId,
```

Comment:
This prop is useless right?

Reply:
Actually, no. It is used by `withDispatch`. It can't be retrieved there as `select` is not available within `withDispatch`, so it needs to be passed (previously it was a component prop passed by the parent).